_sass/_molecules.scss

```diff
@@ -41,16 +39,96 @@
     }
   }
 }
-@mixin foot-cp-nav{
+@mixin flex-end-center {
+  display: flex;
+  justify-content: end;
+  align-items: center;
+}
+
+@mixin hide-burger {
+  & .nav_burger {
+    @media (min-width: $grid-media-size) {
+      display: none;
+    }
+  }
+}
+@mixin main-nav-init-state {
+  @media (min-width: $grid-media-size) {
+    &[data-state="closed"],
+    &[data-state="opened"] {
+      ul {
+        display: flex;
+        align-items: center;
+        flex-direction: row;
+        & li{
+          @include nav-li();
+        }
+      }
+      .nav_burger {
+        display: none;
+      }
+    }
+  }
+}
+@mixin burger-closed {
+  #menu-opened {
+    display: none;
+  }
+  #menu-closed {
+    display: initial;
+  }
+}
+@mixin burger-opened {
+  #menu-opened {
+    display: initial;
+  }
+  #menu-closed {
+    display: none;
+  }
+}
+@mixin main-nav-state {
+  &[data-state="closed"] {
+    ul {
+      display: none;
```

Comment:
To clean this up: Make display: none be the default for `.nav_main .nav_ul`and set a media query to switch to flex layout.

Moreso, extract ul to its own thing. i.e copy and paste `.nav_ul`.